include/nccl_ofi.h

```diff
@@ -331,6 +331,14 @@ struct nccl_net_ofi_ep {
 				nccl_net_ofi_conn_handle_t *handle,
 				nccl_net_ofi_send_comm_t **send_comm);
 
+	/*
+	 * @brief 	Inline write to a peer address
+	 *
+	 * Perform write to peer address.  There is no request because return is immediate.
```

Comment:
Why isn't there a request? The `fi_write` itself will produce a CQ event. How does the caller know when the write has completed?